### PR TITLE
#Updated

### PR DESCRIPTION
**Pull request by kaishengchan-sterlingrisq**
- README Pull Request comment for review

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,5 @@
 # TestCodeReview
 Test Code Review &amp; Pull Request With GitHub From VS
+
+#Kai
```
Comment (faroque-hossain) [inline README.md:4]: Looks good but please also add one more line so that it has more content

**Review by faroque-hossain — CHANGES_REQUESTED** [review]
Hi Kai,
Please check the comment on the readme file.

File: README.md (hunk 1 above)
Comment (kaishengchan-sterlingrisq) [reply README.md:4]: I will add and get back to you

**Review by faroque-hossain — APPROVED** [review]
Well done!

**Pull request merged**